utilities/test_suite/HOST/Tensor_image_host.cpp

```diff
@@ -1630,6 +1630,23 @@ int main(int argc, char **argv)
 
                     break;
                 }
+                case JPEG_COMPRESSION_DISTORTION:
+                {
+                    testCaseName = "jpeg_compression_distortion";
+
+                    Rpp32s qualityTensor[batchSize];
+                    for (i = 0; i < batchSize; i++)
+                        qualityTensor[i] = 50;
```

Comment:
I think it is better to test with a random quality tensor which can have any value in the range